bears/scss/SCSSLintBear.py

```diff
@@ -13,12 +16,297 @@ class SCSSLintBear:
     """
 
     LANGUAGES = {"SCSS"}
-    REQUIREMENTS = {GemRequirement('scss-lint', '', 'false')}
+    REQUIREMENTS = {GemRequirement('scss-lint', '', 'false'),
+                    PipRequirement('pyyaml', '3.*')}
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Syntax', 'Formatting'}
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return filename,
+        return filename, '--config', config_file
+
+    @staticmethod
+    def generate_config(filename, file,
+                        space_before_bang: bool=True,
+                        space_after_bang: bool=False,
```

Comment:
imo in normal conversations you should have a space after bang, not before
``I mean, watch this! This is how a sentence should look like``

im not entirely sure how it is in scss tho